### PR TITLE
update desktop bubble from Advantage to Pro

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

**Don't merge** until Jan 26th
- This PR addressed 4 different issue, all with minor/similar changes to them, to be merged into the feature branch [ubuntu-pro-ga-release](https://github.com/canonical/ubuntu.com/tree/ubuntu-pro-ga-release) on the 26th Jan

## QA

- /desktop/features [copydoc](https://docs.google.com/document/d/1nLHJlZ3igDwAAOUDP03wp4iKLxZts8110EA2ZOF5uaU/edit#)
- /pricing/desktops [copydoc](https://docs.google.com/document/d/1xVtaZ7wr5sdBqye3Gc-XWXEpNqXaol8uDdMwJ_KyOO8/edit) *there are some comments about the table that have been ignored in this PR as it is out of scope*
- /download/desktop [copydoc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit#) 
- /desktop/organisations [copydoc](https://docs.google.com/document/d/1RuxCA6GYh9UriNv26HCZP9UHo4b4_9fYaaMtGQuH_xc/edit#%7Chttps://docs.google.com/document/d/1RuxCA6GYh9UriNv26HCZP9UHo4b4_9fYaaMtGQuH_xc/edit#%7Csmart-link) 

## Issue / Card

Fixes
https://github.com/canonical/web-squad/issues/6337
https://github.com/canonical/web-squad/issues/6340
https://github.com/canonical/web-squad/issues/6339
https://github.com/canonical/web-squad/issues/6329

Jira: https://warthogs.atlassian.net/browse/WD-1225

### CODE REVIEW

File: templates/download/desktop/index.html [hunk 1]
```diff
@@ -239,19 +243,19 @@ <h3>Mini-clouds on desktops with Multipass</h3>
...
-        url="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg",
+        url="https://assets.ubuntu.com/v1/ce518a18-CoF-2022_solid+O.svg",
         alt="",
-        width="140",
-        height="140",
+        width="156",
+        height="150",
         hi_def=True,
-        loading="lazy"
+        loading="auto|lazy"
```
Comment (mtruj013) [inline templates/download/desktop/index.html:251]:
```suggestion
        loading="lazy"
```

File: templates/desktop/organisations.html [hunk 2]
```diff
@@ -600,6 +596,31 @@ <h2>Secure enterprise management with Ubuntu Pro Desktop</h2>
...
 </section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Virtual desktops, powered by AWS</h2>
+      <p>Ubuntu Desktop on Amazon WorkSpaces offers developers an optimised Ubuntu Desktop environment to rapidly build, test and deploy code &mdash; spinning up and tearing down instances as required.</p>
+      <p>IT organisations benefit from rapid deployment, configuration and scalability for their managed Ubuntu Desktops whilst their applications and data remain secured in the AWS cloud, even when their developers are working remotely.</p>
+      <p>
+        <a href="/aws/workspaces">Find out more&nbsp;&rsaquo;</a>
```
Comment (mtruj013) [inline templates/desktop/organisations.html:607]: From an accessibility perspective, this link text needs to be more descriptive. I've left a comment for Oliver on the doc

File: templates/pricing/desktops.html [hunk 3]
```diff
@@ -256,6 +256,10 @@ <h2>Power to developers. Peace of mind for IT. Get Ubuntu Pro Desktop.</h2>
           Optional weekday or 24x7 support tiers
         </li>
       </ul>
+      <p>For more information, download our whitepaper:</p>
+      <p>
+        <a href="engage/adopting-secure-enterprise-linux-desktop" class="p-button">Adopting a secure enterprise Linux desktop</a>
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/pricing/desktops.html:261]: I spoke with Oliver about this, the whitepaper will go live the same day we releases these changes aka. Jan 26th

File: templates/desktop/organisations.html (hunk 2 above)
Comment (petesfrench) [reply templates/desktop/organisations.html:607]: Oliver suggest 'Find out more about Ubuntu WorkSpaces'

**Review by mtruj013 — APPROVED** [review]
LGTM, thanks @petesfrench !

**Pull request merged**